src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        String str = new ColorSupplier().getRandomColor();
+        int index = new Random().nextInt(100) + 1;
```

Comment:
make ColorSupplier (and Random) class field, not a local variable
make 100 a class constant
rename uninformative `str` to `color`